app/move/views/view.njk

```diff
@@ -16,7 +16,7 @@
   {{ govukBackLink({
     text: t("actions::back_to_dashboard"),
     classes: "app-print--hide",
-    href: "/moves"
+    href: "/moves" + MOVES_QUERY_STRING
```

Comment:
You might consider allowing the middleware to specify the whole path including string. In general I would recommend avoiding paths being hard coded or constructed in the view.